src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        final int Number = 3;
+        final int Range = 100;
```

Comment:
The constants `Number` and `Range` should be named using uppercase letters with underscores as separators, following Java naming conventions for constants. Consider renaming them to `NUMBER` and `RANGE`.